samples/openapi3/client/features/dynamic-servers/python/dynamic_servers/api_client.py

```diff
@@ -231,6 +235,26 @@ def __call_api(
             return (return_data, response_data.status,
                     response_data.getheaders())
 
+    def parameters_to_multipart(self, params, collection_types):
+        """Get parameters as list of tuples, formatting as json if value is collection_types
+
+        :param params: Parameters as dict or list of two-tuples
```

Comment:
If we keep your code as-is, then it will always ingest tuples for params.
So if you want to keep it as-is, please change this docstring to describe params as tuples not a dict.